closure_externs.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-disable */
+
+/**
+ * @fileoverview This file is where any externs for third party libraries we use should be placed
+ * so that closure can type-check them.
+ *
+ * Closure historically works using "namespaces", e.g.
+ *
+ * // fileA.js
+ * goog.provide('app.a');
+ * app.a.return5 = () => 5;
+ *
+ * // fileB.js
+ *
+ * goog.provide('app.b');
+ * goog.require('app.a');
+ * app.b.return10 = () => app.a.return5() * 2;
+ *
+ * The way that closure reconciles this with path-based module loading systems
+ * such as CommonJS and ES2015 is by using the `goog:` prefix to load any `goog.provide()`'d
```

Comment:
I prefer you explicitly said in the comment. It might be less fancier but make things clear.